pkg/ztunnel/xds/stream_processor.go

```diff
@@ -232,29 +285,8 @@ func (sp *StreamProcessor) Start() {
 					logfields.Error,
 					err,
 					logfields.EndpointID,
-					epEvent.ID)
+					epEvent.UID)
```

Comment:
Not part of this PR's diff, but comment on line 229 "// new event from endpoint manager" needs to be updated, since we are getting events from k8s watcher now.